### DISCUSSION TITLE
Thanks for this library

### DISCUSSION THREAD

**Discussion by ArulselvanMadhavan** [datadog4s/datadog4s#369] [General]
I'm currently introducing this library in a second project at work and could not be happier with the design, docs and the ease with which I can get started. Great work! I hope to see this become well adopted and widely used. Cheers

**Comment by tomasherman** [comment]
Cheers mate! That made my day, thanks for sharing. It's great to hear that you enjoy using this lib and makes your work-life a little easier. Please let me know if there are some ways you can think of that would make the lib even better.

Also, i would love to hear what projects you work on and for which company :)

**Reply by ArulselvanMadhavan** [reply]
Of course! I'd be happy to contribute to this library. I noticed a couple of things in the documentation today when I was trying to add a timer. 

```scala
    val nanoTimer = factory.timer.distribution("nano-timer", timeUnit = TimeUnit.NANOSECONDS)
```

It looks like the second argument is no longer a timeUnit. I'm using the latest version of this library

And, BTW, I work for a company called Chartboost at San Francisco. The first project I used this library was a heavily used microservice. We had some legacy DD metering that I didn't quite like and wasn't working very well for metrics aggregration(Veneur) and I used this library to get metrics aggregation work smoothly for us.

Currently, I'm adding your library on a machine learning model training pipeline. So, it seems like the data scientists can understand IO and effect based systems as long as they have good samples to follow

**Reply by tomasherman** [reply]
Ah that's because The version with that support is not yet released, only on snapshot version. It was merged only s few days ago. But I can release it later today

**Reply by tomasherman** [reply]
@ArulselvanMadhavan just FYI `0.13.0` version is out and adds the mentioned feature. Hope it will work well for ya ;)

**Reply by ArulselvanMadhavan** [reply]
Thanks for letting me know! I'm looking forward to trying it

**Reply by ArulselvanMadhavan** [reply]
Tried it for few days in production and the time unit feature is pretty neat, and removes some ugly math we were doing in DD dashboards

**Reply by tomasherman** [reply]
Awesome! Really happy you like it and saves you some work
